src/simbridge-client/src/components/Health.ts

```diff
@@ -0,0 +1,39 @@
+import { simbridgeUrl } from '../common';
+
+/**
+ * Class responsible for retrieving data related to company routes from SimBridge
+ */
+export class Health {
+    /**
+     * Used to check the state of a given service. If none is given, then main status is returned.
+     * @param serviceName The name of the service or omit for the overall status ('' || 'api || 'mcdu')
+     * @returns true if service is available, false otherwise
+     */
+    public static async getHealth(serviceName: String = ''): Promise<boolean> {
```

Comment:
Give serviceName a better, more specific type than `String` with a union. Ex: `'api' | 'mcdu'` Also instead of the default being `''` you could just change the signature to use `serviceName?: 'api' | 'mcdu'` and replace `case '':` with `case undefined:`

Reply:
Done - always forget that JS has these :)